edx_proctoring/templates/timed_exam/ready_to_submit.html

```diff
@@ -2,24 +2,23 @@
 <div class="sequence timed-exam completed" data-exam-id="{{exam_id}}">
   <h3>
     {% blocktrans %}
-      Are you sure you want to end your timed exam?
+      Are you sure that you want to end your timed exam?
     {% endblocktrans %}
   </h3>
   <p>
     {% blocktrans %}
-      Make sure your responses and work are ready to be submitted. Once they are, you may end the exam below
-      and your worked will then be graded.
+      Make sure your answers are ready to be submitted. When you are ready, submit the exam below. Your exam will then be graded.
```

Comment:
This is just a little redundant. Maybe "Make sure your answers are ready to be submitted, and then submit your exam. Your exam will then be graded."